templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +28,30 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
+        BOT_IDENTITY = {  # Use the corresponding values in your bot's `.zuliprc`
           'email': '<err-bot@your.zulip.server>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
           'site': '<http://your.zulip.server>'
         }
         BOT_ADMINS = ('<your@email.address',)
         CHATROOM_PRESENCE = ()
-        BOT_PREFIX = '<@**err-bot**>'  # Providing errbot's full name in Zulip lets it respond to @-mentions.
+        BOT_PREFIX = '<@**err-bot**>'  # Providing your errbot's full name
+        # in Zulip lets it respond to @-mentions.
 
     Sections you need to edit are marked with `<>`.
 
-1. [Start ErrBot](https://errbot.readthedocs.io/en/latest/user_guide/setup.html#starting-the-daemon).
+1. [Start ErrBot][start-errbot].
+
+!!! tip ""
+
+    Rooms in ErrBot are channels in Zulip.
+
+{end_tabs}
 
 {!congrats.md!}
 
 ![Errbot message](/static/images/integrations/errbot/000.png)
```

Comment:
Sigh, another screenshot with an old UI. Whatever we decide for Capistrano can probably be extended to this one and any similar ones.